packages/react-test-renderer/src/ReactTestRenderer.js

```diff
@@ -461,7 +461,15 @@ const ReactTestRendererFiber = {
         if (container.children.length === 1) {
           return toJSON(container.children[0]);
         }
-
+        if (
+          container.children.length === 2 &&
+          container.children[0].isHidden === true &&
```

Comment:
Can we throw if we encounter .isHidden in any other context?

Reply:
What do you mean by "in any other context"?

Reply:
If I understand right, today we only use .isHidden for the two children of a Suspense component. If we ever change that we may want to update this logic in the test renderer code. So I thought it might be smart to assert that !isHidden except in this specific case. (For example, if arbitrary user code could make host components hidden today, then this logic would incorrectly catch those if there's ever two children with the first child hidden.)

Reply:
> For example, if arbitrary user code could make host components hidden today

Ah. This `isHidden` attribute is set by the renderer's config though. It's not something users can directly set. So if we ever had an invalid `isHidden` setting, that would mean a bug in React (or at least the test renderer).

Reply:
I'm trying to catch the case where we change React and forget about the test renderer.

Reply:
I guess that makes sense...but it feels like that would be a kind of heavyweight way to guard against that.

I think we would need to update this function, as well as the `toJSON` recursive method it calls, to check every child for `isHidden === true` and if it ever finds them– verify that there is only one sibling, and that it's `isHidden === false`.